src/pages/components/index.html

```diff
@@ -4,7 +4,22 @@ <h1 class="title">Components</h1>
     </div>
 
     <div class="row text input">
-        <!-- Text input here -->
+        
+        
+            <div class = "text-input-box">
+                
+                <h3>
+                    Inputs
+                </h3>
+                <p>Normal</p>
+                
+                <div class="forms-input mb-3">
+                    <input class="input-box" type="text" placeholder="First name">
+                
+                </div>
+            </div>
+               
+       
```

Comment:
Need to have a pattern, lots of spaces needed

Reply:
done!